src/Platform/Microsoft.Testing.Platform/OutputDevice/ConsoleDisplayService.cs

```diff
@@ -144,7 +145,17 @@ public async Task DisplayBannerAsync()
                     var version = (AssemblyInformationalVersionAttribute?)Assembly.GetExecutingAssembly().GetCustomAttribute(typeof(AssemblyInformationalVersionAttribute));
                     if (version is not null)
                     {
-                        stringBuilder.AppendLine(CultureInfo.InvariantCulture, $"Version: {version.InformationalVersion}");
+                        stringBuilder.Append(CultureInfo.InvariantCulture, $"Version: {version.InformationalVersion}");
+
+                        var buildTime = Assembly.GetExecutingAssembly().GetCustomAttributes(typeof(AssemblyMetadataAttribute))
+                        .OfType<AssemblyMetadataAttribute>().FirstOrDefault(x => x.Key == BUILDTIME_ATTRIBUTE_NAME);
+                        if (buildTime is not null)
+                        {
+                            DateTime buildDateTime = DateTime.FromOADate(double.Parse(buildTime.Value!, CultureInfo.InvariantCulture));
+                            stringBuilder.Append(CultureInfo.InvariantCulture, $" ({buildDateTime.ToString("MM/dd/yy", CultureInfo.InvariantCulture)})");
```

Comment:
Shall we use locale preference for the date display? If not, I would maybe go for the YYYY/MM/dd format as it's less confusing globally.

Reply:
I think that (UTC  YYYY/MM/dd) is good

Reply:
<img width="330" alt="image" src="https://github.com/microsoft/testfx/assets/7894084/d1fc2244-233e-46f2-87ab-61f37d922f8f">

Reply:
No no it's good and makes it explicit